cli/tests/e2e/rules/regex-nosemgrep.yaml

```diff
@@ -2,6 +2,10 @@ rules:
   - id: detected-aws-account-id
     pattern-regex: |-
       ("|')?(AWS|aws|Aws)?_?(ACCOUNT|account|Account)_?(ID|id|Id)?("|')?\s*(:|=>|=)\s*("|')?[0-9]{12}("|')?
+    metavars:
+      - $1: $SHOULDNTEXIST
+      - $2: $AWS
```

Comment:
I would just write this as `- 2: $AWS`. We're mapping capture group 2 to metavariable `$AWS`. The idea is that there will no be any metavariable `$2`.